tailwind.config.js

```diff
@@ -18,6 +18,9 @@ module.exports = {
       purple: {
         DEFAULT: '#9747FF',
       },
+      darkMagenta: {
+        DEFAULT: '#7D4B93',
+      },
```

Comment:
Enter this color inside the magenta object as 600, 700 or 800
```suggestion
      magenta: {
        700: '#7D4B93',
      },
```